File: Exemplos SmartPOS/App_IntentDigitalHUB_Java/Elgin_IntentDigitalHUB_SmartPOS_Java-font/app/src/main/java/com/elgin/java_intentdigitalhub_smartpos/IntentDigitalHubService/ELGINPAY/Commands/SetPersonalizacao.java
package com.elgin.java_intentdigitalhub_smartpos.IntentDigitalHubService.ELGINPAY.Commands;

import com.elgin.java_intentdigitalhub_smartpos.IntentDigitalHubService.ELGINPAY.ElginPayCommand;
import com.google.gson.JsonObject;

public class SetPersonalizacao extends ElginPayCommand {
    final private String iconeToolbar;
    final private String fonte;
    final private String corFonte;
    final private String corFonteTeclado;
    final private String corFundoToolbar;
    final private String corFundoTela;
    final private String corTeclaLiberadaTeclado;
    final private String corFundoTeclado;
    final private String corTextoCaixaEdicao;
    final private String corSeparadorMenu;


    public SetPersonalizacao(String iconeToolbar, String fonte, String corFonte, String corFonteTeclado, String corFundoToolbar, String corFundoTela, String corTeclaLiberadaTeclado, String corFundoTeclado, String corTextoCaixaEdicao, String corSeparadorMenu) {
        super("setPersonalizacao");
        this.iconeToolbar = iconeToolbar;
        this.fonte = fonte;
        this.corFonte = corFonte;
        this.corFonteTeclado = corFonteTeclado;
        this.corFundoToolbar = corFundoToolbar;
        this.corFundoTela = corFundoTela;
        this.corTeclaLiberadaTeclado = corTeclaLiberadaTeclado;
        this.corFundoTeclado = corFundoTeclado;
        this.corTextoCaixaEdicao = corTextoCaixaEdicao;
        this.corSeparadorMenu = corSeparadorMenu;
    }

    @Override
    protected JsonObject functionParametersJson() {
        JsonObject functionParametersJson = new JsonObject();

        functionParametersJson.addProperty("iconeToolbar", this.iconeToolbar);
        functionParametersJson.addProperty("fonte", this.fonte);
        functionParametersJson.addProperty("corFonte", this.corFonte);
        functionParametersJson.addProperty("corFonteTeclado", this.corFonteTeclado);
        functionParametersJson.addProperty("corFundoToolbar", this.corFundoToolbar);
        functionParametersJson.addProperty("corFundoTela", this.corFundoTela);
        functionParametersJson.addProperty("corTeclaLiberadaTeclado", this.corTeclaLiberadaTeclado);
        functionParametersJson.addProperty("corFundoTeclado", this.corFundoTeclado);
        functionParametersJson.addProperty("corTextoCaixaEdicao", this.corTextoCaixaEdicao);
        functionParametersJson.addProperty("corSeparadorMenu", this.corSeparadorMenu);

        return functionParametersJson;
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/ELGINPAY/Commands/set_personalizacao.dart
import '../elginpay_command.dart';

class SetPersonalizacao extends ElginPayCommand {
  final String iconeToolbar;
  final String fonte;
  final String corFonte;
  final String corFonteTeclado;
  final String corFundoToolbar;
  final String corFundoTela;
  final String corTeclaLiberadaTeclado;
  final String corFundoTeclado;
  final String corTextoCaixaEdicao;
  final String corSeparadorMenu;

  SetPersonalizacao(
      this.iconeToolbar,
      this.fonte,
      this.corFonte,
      this.corFonteTeclado,
      this.corFundoToolbar,
      this.corFundoTela,
      this.corTeclaLiberadaTeclado,
      this.corFundoTeclado,
      this.corTextoCaixaEdicao,
      this.corSeparadorMenu)
      : super('setPersonalizacao');

  @override
  get functionParametersJson => {
        'iconeToolbar': iconeToolbar,
        'fonte': fonte,
        'corFonte': corFonte,
        'corFonteTeclado': corFonteTeclado,
        'corFundoToolbar': corFundoToolbar,
        'corFundoTela': corFundoTela,
        'corTeclaLiberadaTeclado': corTeclaLiberadaTeclado,
        'corFundoTeclado': corFundoTeclado,
        'corTextoCaixaEdicao': corTextoCaixaEdicao,
        'corSeparadorMenu': corSeparadorMenu,
      };
}
